### ISSUE TITLE
Email and Github form fields are not sending me verification emails or links

### ISSUE THREAD

**Issue by michaelsena** [uport-project/3box-dapp#60]
**Describe the bug**
4 submissions made via the provided form. No unique links/ids were provided by said form to verify submissions; no messages received on the provided email.

**To Reproduce**
1. Go to edit profile page
2. Enter details
3. Click submit
4. Nothing happens

**Expected behavior**
I expected to see verification emails or links generated from this claim.

**Additional context**
This isn't really a bug on our part, but it seems like one to users. We should consider when we want to add verification. Maybe this would be a good feature to bounty? @oed @KamesCG

**Comment by oznekenzo** [comment]
@michaelsena users are supposed to get verification emails after entering an email?  Not quite sure what's meant by "No unique links/ids were provided by said form".  Is this a bug?  I don't think we ever went over behavior as explained above 😅but down to get started on this!!

**Comment by oed** [comment]
@michaelsena this needs some more thought before we implement it. The main problem is "how do we ensure that the email verification is not fakeable" i.e. I use 3box-js directly to add it. It's solvable, but I don't think it should be a prio rn.

**Comment by michaelsena** [comment]
Agreed I was just about to type that. I added this not as a bug but as feedback that we’ll eventually get to. @oznekenzo let’s prio web3 login and performance and mobile issues that affect people not being able to use the dapp. Then we’ll get to missing features.
